int32 num_robot
int32 position
int32 duree
